notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -394,7 +363,7 @@
       "source": [
         "### Create BigQuery client\n",
         "\n",
-        "In this tutorial, you explore the monitoring data stored in BigQuery. You create a client interface, which you subsequently use to access the data."
+        "In this tutorial, you use data from the public BigQuery table. You create a client interface, which you subsequently use to access the data."
```

Comment:
Change "subsequently" to "then"

- You create a client interface, which you then use to access the data.